osbuild-composer.spec

```
@@ -237,6 +249,28 @@ systemctl stop "osbuild-worker@*.service" "osbuild-remote-worker@*.service"
 # restart all the worker services
 %systemd_postun_with_restart "osbuild-worker@*.service" "osbuild-remote-worker@*.service"
 
+%if 0%{?rhel}
+%package osbuild-composer-cloud
```

This (and all the other usages of `cloud`) should just be:

```suggestion
%package cloud
```

`%package` macro by default adds the "main" package as a prefix. See the following examples:

`%package osbuild-composer-cloud` => `osbuild-composer-osbuild-composer-cloud`
`%package cloud` => `osbuild-composer-cloud`

If this behaviour is not desired, one must use `-n` argument:

`%package -n osbuild-composer-cloud` => `osbuild-composer-cloud`
`%package -n cloud` => `cloud`

This is used e.g. in `lorax.spec` with `composer-cli` subpackage:
https://github.com/weldr/lorax/blob/52fd0f823b27c157f28a5900b7899fc02dcd3559/lorax.spec#L171